NeoFlux/Controllers/ContractController.cs

```diff
@@ -149,5 +160,37 @@ public JsonResult Transfer([FromBody] JObject jsonData)
                 return null;
             });
         }
+        
+        [HttpPost]
+        [Route("get_transaction_info")]
+        [Authorize]
+        public JsonResult GetBestBlock([FromBody]JObject jsonData)
+        {
+            try
+            {              
+                var txId = jsonData.GetValue("txId").Value<string>();
+                var response = RequestUtils.Request(RequestType.GET,

```

Comment:
Why not make all this a class, and use a factory to get it, so you can keep the python node encapsulated in there?